client/blocks/sharing-preview-pane/utils.js

```diff
@@ -56,3 +56,17 @@ export const getExcerptForPost = post => {
 		)
 	);
 };
+
+export const getSummaryForPost = ( post, translate, maxWords = 60 ) => {
```

Comment:
@Copons do they normally truncate by words or approximate character length?

Reply:
Did some testing in FB, and looks like it's a bit arbitrary. ~240 characters for single word chars and ~500ish for longer words.

Let's leave some JSDoc that this function is an approximation of the truncation logic of what our servers sent to FB, and that it will be out of sync with that and FB's actual truncation algorithm.

If there are specific truncation cases we'd like to cover, lets add unit tests here.